package.json

```diff
@@ -20,17 +20,20 @@
   "license": "ISC",
   "devDependencies": {
     "@faker-js/faker": "7.4.0",
+    "chai": "^4.3.6",
     "eslint": "8.16.0",
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-mocha-no-only": "^1.1.1",
```

Comment:
@beblicarl please what role does this play

Reply:
It ensures describe and it are not flagged

Reply:
okay, still flagging at my end, can you share your reference to what it does?

Reply:
`https://github.com/lewazo/eslint-mocha-no-only`

Reply:
https://github.com/lewazo/eslint-mocha-no-only

Reply:
yea makes sense why it will not flag it but actually does something more that we're not looking at yet. adding `mocha: true` as shown here should work for this case https://stackoverflow.com/a/50111712

Reply:
Alright thanks